index.html

```diff
@@ -17,219 +28,358 @@
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <img class="b-img-1" alt="Background Image" src="assets/Illustration.png">
+            </div>
+        </div>
+        <img class="desktop-b-img-1" src="assets/IllustrationDesktop.png" alt="Background Image">
 
-<section class="recent-works">
-    <div class="first">
-        <h3>My Recent Works</h3>
-        <hr>
-    </div>
-    <div class="recent-work" id="recent-works">
-        <a href="" class="women"><img src="assets/Img_Placeholder.png" alt="women doing yoga"></a>
-    <div>
-        <h4>Multi-Post-Stories</h4>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        <ul class="tech-used">
-            <li class="css">Css</li>
-            <li class="HTML">html</li>
-            <li class="boots">Bootstrap</li>
-            <li class="ruby">Ruby</li>  
-        </ul>
-        <button type="button" >See Project</button> 
-    </div>
-    <!-- first-Profesional-Art-Printing-Data -->
-    <div class="card-section">
-        <div class="work-card">
-            <div class="card-content flex-container">
-                <h4 class="title">Profesional Art Printing Data</h4>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <ul>
-                    <li class="html">html</li>
-                    <li class="Boot">Bootstrap</li>
-                    <li class="Ruby">Ruby</li>
+    </section>
+    <section class="my-projects">
+        <div class="projects-subject">
+            <h1 class="projects">My Recent Works</h1>
+            <hr class="line">
+        </div>
+        <div class="project-1">
+            <img alt="Project Image" class="project-img" src="assets/Img_Placeholder.png">
+            <div class="project-head">
+                <h2 class="heading2 multy-task">Multi-Post Stories</h2>
+                <p class="project-discription">A daily selection of privately personalized reads; no accounts or
+                    sign-ups
+                    required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                    took
+                    a standard dummy text.</p>
+                <ul class="web-icons icons-desktop">
+                    <li>
+                        <img alt="Web Icon" class="icon-web" src="assets/icons/css.png">
+                    </li>
+                    <li>
+                        <img alt="Web Icon" class="icon-web" src="assets/icons/html.png">
+                    </li>
+                    <li>
+                        <img alt="Web Icon" class="icon-web" src="assets/icons/bootstrap.png">
+                    </li>
+                    <li>
```

Comment:
- [ ] You have too much spacing in between the image and content kindly make it as Figma design. On the left side, there is no spacing after the content kindly make spacing by seeing the design
- [ ] kindly make button at left side as shown in Figma design
### your design
![his](https://user-images.githubusercontent.com/81727988/173112463-ae976ef7-1e39-4bd0-b8c3-dd481016e7cb.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/173112629-efec1edf-5f94-4b09-800b-ca66e9ae587b.png)